File: empty-spring-boot/src/main/scala/com/example/demo/DemoApplication.scala
package com.example.demo

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class DemoApplication

object DemoApplication {

  def main(args: Array[String]) :Unit = {
    SpringApplication.run(classOf[DemoApplication], args: _*)
  }
}

File: group-hystrix/hystrix-001/src/main/java/com/example/my/DemoApplication.java
package com.example.my;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ConfigurableApplicationContext;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
//@EnableScheduling
//@ComponentScan("cn.futuremove.tsp.uni.config.dbs")
public class DemoApplication {

    private static final Logger LOGGER = LoggerFactory.getLogger(DemoApplication.class);


	public static void main(String[] args) {

        ConfigurableApplicationContext context = SpringApplication.run(DemoApplication.class, args);
        LOGGER.info("ConfigurableApplicationContext : {}", context);

    }
}
